client/options.go

```diff
@@ -247,6 +248,110 @@ func IsBackup() BulkInsertOption {
 	}
 }
 
+func WithBucket(bucket string) BulkInsertOption {
+	return func(req *milvuspb.ImportRequest) {
+		optionMap := entity.KvPairsMap(req.GetOptions())
+		optionMap["bucket"] = bucket
+		req.Options = entity.MapKvPairs(optionMap)
+	}
+}
+
+func WithStorageType(storageType string) BulkInsertOption {
+	return func(req *milvuspb.ImportRequest) {
+		optionMap := entity.KvPairsMap(req.GetOptions())
+		optionMap["storage_type"] = fmt.Sprint(storageType)
+		req.Options = entity.MapKvPairs(optionMap)
+	}
+}
+
+func WithAddress(address string) BulkInsertOption {
+	return func(req *milvuspb.ImportRequest) {
+		optionMap := entity.KvPairsMap(req.GetOptions())
+		optionMap["endpoint"] = address
+		req.Options = entity.MapKvPairs(optionMap)
+	}
+}
+
+func WithAccessKeyID(accessKeyID string) BulkInsertOption {
+	return func(req *milvuspb.ImportRequest) {
+		optionMap := entity.KvPairsMap(req.GetOptions())
+		optionMap["access_kye_id"] = accessKeyID
+		req.Options = entity.MapKvPairs(optionMap)
+	}
+}
+
+func WithSecretAccessKeyID(secretAccessKeyID string) BulkInsertOption {
+	return func(req *milvuspb.ImportRequest) {
+		optionMap := entity.KvPairsMap(req.GetOptions())
+		optionMap["secret_access_key_id"] = secretAccessKeyID
```

Comment:
I suggest aligning the parameter names here with the parameter names in milvus.yaml, except for start_ts and end_ts. These two parameters（start_ts, end_ts） are only used by the backup tool, so it is acceptable to have inconsistent naming styles.

![image](https://github.com/milvus-io/milvus-sdk-go/assets/59249785/e25f4df0-4e39-4ab2-8aaf-38064d02f086)